tasks.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import os
+import subprocess
+import sys
+
+from invoke import task
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(levelname)s - %(message)s')
+
+APP_NAME = 'envars'
+PYTHON_VERSION = 3.10
+
+
+def confirm(message):
+    print(message)
+    choice = input().strip().lower()
+    return choice == 'y'
+
+
+def get_branch_name():
+    branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
+    print(f"Current branch name: {branch_name}")
+    return branch_name
+
+
+def get_python_version(file_path='PYTHON_VERSION'):
+    if not os.path.exists(file_path):
+        logging.info(f'Error: File {file_path} does not exist')
+        return PYTHON_VERSION
+
+    try:
+        with open(file_path, 'r') as f:
+            version = f.read().strip()
+            if not version:
+                logging.info('Error: File is empty')
+                return None
+
+            return version
+    except Exception as e:
+        logging.info(f'An error occurred while reading file: {e}')
+        return None
+
+
+@task
+def python_deps_upgrade(c):
+    """Update pip requirements"""
+    c.run('pip-compile --resolver=backtracking -U')
+    c.run('pip install -r requirements.txt')
+
+
+@task
+def python(c):
```

Comment:
python and clean can be used from timeout-tools no need to include in every repo tasks.py

Reply:
I don't use timeout-tools as I don't like the multi folder workflow.  

Do you just want me to delete python and clean?